### PR TITLE
DOC: style main page card

### PR DESCRIPTION
**Pull request by tupui**
Update the CSS of the cards on the main page of the documentation. The current panel has very large images.

Left is current, right is this PR–same zoom level was used.

currnew

### CODE REVIEW

**Comment by tupui** [comment]
If it's preferred I can also go back to this padding. Note the gray area.

```css
.intro-card {
  padding: 20px 0px 20px 0px;
  margin: 10px;
}
```
alt

**Comment by seberg** [comment]
I don't have a strong opinion, maybe a slight tendency to restoring the no-border version.  I feel that the current pictograms are oversized so I prefer the scaled down version (but don't care much how much we scale them down).

Ping @MarsBarLee again, IIRC, you took part in styling this?  Do you have any opinions/preferences here?

File: doc/source/_static/numpy.css [hunk 1]
```diff
@@ -32,3 +31,24 @@ h3 {
   color: #013243; /* warm black */
   letter-spacing: -.03em;
 }
+
+.intro-card {
+  padding: 20px 10px 20px 10px;
+  margin: 10px;
+}
```
Comment (tupui) [inline doc/source/_static/numpy.css:38]: For the no-border on the gray.
```suggestion
.intro-card {
  padding: 20px 0px 20px 0px;
  margin: 10px;
}
```

**Review by rossbar — APPROVED** [review]
This LGTM, thanks @tupui !

This was discussed a bit at the docs meeting and the consensus was that this is an improvement

**Comment by tupui** [comment]
Thanks Ross!

**Pull request merged**